placelist.php

```diff
@@ -218,28 +199,27 @@
 
 		echo '<div class="loading-image"></div>';
 		echo '<div id="places-tabs"><ul>';
-		if ($myindilist) {
+		if (!empty($myindilist)) {
 			echo '<li><a href="#places-indi"><span id="indisource">', I18N::translate('Individuals'), '</span></a></li>';
 		}
-		if ($myfamlist) {
+		if (!empty($myfamlist)) {
 			echo '<li><a href="#places-fam"><span id="famsource">', I18N::translate('Families'), '</span></a></li>';
 		}
 		echo '</ul>';
-		if ($myindilist) {
+		if (!empty($myindilist)) {
 			echo '<div id="places-indi">', FunctionsPrintLists::individualTable($myindilist), '</div>';
 		}
-		if ($myfamlist) {
+		if (!empty($myfamlist)) {
 			echo '<div id="places-fam">', FunctionsPrintLists::familyTable($myfamlist), '</div>';
 		}
-		if (!$myindilist && !$myfamlist) {
+		if (empty($myindilist) && empty($myfamlist)) {
```

Comment:
There isn't a corresponding tab for this block.

However, I'm not sure this code branch is even needed.  Surely, there must be at least one individual or one family - otherwise where did the place come from?